src/MartinGeorgiev/Doctrine/ORM/Query/AST/Functions/BaseFunction.php

```diff
@@ -31,7 +31,22 @@ abstract class BaseFunction extends FunctionNode
      */
     protected array $nodes = [];
 
-    abstract protected function customizeFunction(): void;
+    /**
+     * TODO Make abstract when {@see customiseFunction()} is no more
+     */
+    protected function customizeFunction(): void
```

Comment:
Leaving this abstract won't work, because none of the classes currently overriding the old name will be overriding the new name, and so adding the old method alone isn't enough.